lib/hammer_cli_katello/local_helper.rb

```diff
@@ -0,0 +1,12 @@
+module HammerCLIKatello
+  module LocalHelper
+    def local_only
+      return true if system("rpm -q foreman", :out => File::NULL)
```

Comment:
@akofink In the tests, I can't figure out how to either mock `system` or mock the overall call to `local_only`. I have tried many a thing with no luck.

Reply:
It'd be easy to stub if the system call was in its own (private) method, and it'd be more descriptive imo.

Reply:
That method could also use better method that'd be compatible with non-RPM systems. Perhaps check for the existence of `/usr/share/foreman` for example.